### PR TITLE
Feature 97 async dropscope

### PR DESCRIPTION
**Pull request by akagr**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Adds ability to use observable return value if dropscope is a fuction. Updates readme. 

* **What is the current behavior?** (You can also link to an open issue here)

Implements #97

### CODE REVIEW

**Comment by akagr** [comment]
@ObaidUrRehman can you take a look at this? I've updated the README.md with observable instructions under *Drop scope as Functions* section.

File: src/directives/droppable.directive.ts [hunk 1]
```diff
@@ -209,10 +209,10 @@ export class Droppable implements OnInit, OnDestroy {
             this.unbindDragEnterListener();
         }
         if (this.unbindDragOverListener) {
-            this.unbindDragEnterListener();
+            this.unbindDragOverListener();
         }
```
Comment (akagr) [inline src/directives/droppable.directive.ts]: This actually took the most amount of time to track down :) Drag over and leave events were getting subscribed to but never being unsubscribed.

Comment (ObaidUrRehman) [reply src/directives/droppable.directive.ts]: Good Eye man!

**Comment by ObaidUrRehman** [comment]
Thanks @akagr

**Pull request merged**